docs/index.rst

```diff
@@ -70,6 +73,7 @@ communicate to them.
 
    tutorials
    explanation
+   timingmodels
```

Comment:
Can this go under "reference"? Or "explanation"? I'd really like to keep the top-level division into tutorials/explanation/reference/how-to.